cocotb/scheduler.py

```diff
@@ -184,11 +184,11 @@ class Scheduler:
         - Any pending writes are cached and do not happen immediately
 
     ReadOnly mode
-        - Corresponds to :any:`cbReadOnlySynch` (VPI) or :any:`vhpiCbLastKnownDeltaCycle`
+        - Corresponds to ``cbReadOnlySynch`` (VPI) or ``vhpiCbLastKnownDeltaCycle``
```

Comment:
Update these to the correct callbacks while you're here.
```suggestion
        - Corresponds to ``cbReadOnlySynch`` (VPI) or ``vhpiCbRepEndOfTimeStep``
```